app/feature/feature-addon-purchase/src/main/kotlin/com/hedvig/android/feature/addon/purchase/data/GetInsuranceForTravelAddonUseCase.kt

```diff
@@ -41,21 +42,12 @@ internal class GetInsuranceForTravelAddonUseCaseImpl(
           }
           raise(ErrorMessage())
         } else {
-          // TODO: remove mock!
-//          val result = memberResponse.bind().currentMember.toInsurancesForAddon(ids)
-//          ensure(result.isNotEmpty()) {
-//            { "Tried to get list of insurances for addon purchase but the list is empty!" }
-//            ErrorMessage()
-//          }
-//          result
-          listOf<InsuranceForAddon>(
-            InsuranceForAddon(
-              id = "id",
-              displayName = "Rent Bas",
-              "Tulegatan 1",
-              ContractGroup.RENTAL,
-            ),
-          ) // TODO: remove mock!!
+          val result = memberResponse.bind().currentMember.toInsurancesForAddon(ids)
+          ensure(result.isNotEmpty()) {
+            { "Tried to get list of insurances for addon purchase but the list is empty!" }
+            ErrorMessage()
```

Comment:
```suggestion
            logcat { "Tried to get list of insurances for addon purchase but the list is empty!" }
            ErrorMessage()
```
Probably missed the logcat here. Otherwise it looks like a lambda which does nothing?
Also would be good to have something inside the ErrorMessage too, so that we can  track the error down easier. But I suppose we can't do this here since this was the place where we relied on the message being there to let them redirect to the chat right?